### PR TITLE
refactor: app flow

### LINKED ISSUE #356: Create user tab
Such as in organization, an user tab should be created.
image

What needs to be done:

- [ ] Create a sidebar when tab 'user' is selected where the following pages are:

	All the pages below should have 2 types of view when an organization tab is selected or when an user tab is selected.
	1.  Home page which has tabs "Following" and "All" where you can get all information related to organizations that you follow or all of them depending on which is selected.
	2.  Activities page  which has tabs "Following" and "All" where you can get all information related to organizations that you follow or all of them depending on which is selected.
	3. Calendar with all information related to activities of organizations that you follow.
	4. Organizations page as it is now.
	5. News page which has tabs "Following" and "All" where you can get all information related to organizations that you follow or all of them depending on which is selected.

- [ ] Change routes to be possible to access those pages without a current_organization.
- [ ] Logged out users have access to activities pages with all organizations information.

This was already started in this [PR](https://github.com/cesium/atomic/pull/316), so in order to solve this you can commit to that PR or create your own.

### PR DESCRIPTION
**Pull request by ruilopesm**
Some pages are not working (or even do not exist) on purpose. After merging this PR, I'll open a bunch of issues 🙏🏼

### CODE REVIEW

**Comment by danielsp45** [comment]
I'm having an issue with the organisations SVG:
image

**Comment by ruilopesm** [comment]
> I'm having an issue with the organisations SVG: image

It should be fixed now 🙏🏼
@danielsp45

File: lib/atomic/departments.ex [hunk 1]
```diff
@@ -30,8 +30,11 @@ defmodule Atomic.Departments do
       [%Department{}, ...]
 
   """
-  def list_departments_by_organization_id(id) do
-    Repo.all(from d in Department, where: d.organization_id == ^id)
+  def list_departments_by_organization_id(id, opts \\ []) do
+    Department
+    |> apply_filters(opts)
+    |> where([d], d.organization_id == ^id)
```
Comment (ruilopesm) [inline lib/atomic/departments.ex]:
```suggestion
    |> where([d], d.organization_id == ^id)
    |> apply_filters(opts)  
```

File: lib/atomic_web/live/department_live/show.ex [hunk 2]
```diff
@@ -1,47 +1,44 @@
 defmodule AtomicWeb.DepartmentLive.Show do
+  alias Atomic.Organizations.Collaborator
   use AtomicWeb, :live_view
 
+  alias Atomic.Accounts
   alias Atomic.Departments
```
Comment (ruilopesm) [inline lib/atomic_web/live/department_live/show.ex]:
```suggestion
   use AtomicWeb, :live_view
 
   alias Atomic.Accounts
   alias Atomic.Departments
   alias Atomic.Organizations.Collaborator
```

File: lib/atomic_web/live/department_live/show.html.heex [hunk 3]
```diff
@@ -55,18 +58,19 @@
...
-                <h3 class="mt-6 text-md font-bold font-medium text-gray-900"><%= session.activity.title %></h3>
+                <h3 class="mt-6 text-md font-medium text-zinc-900"><%= session.activity.title %></h3>
               </div>
               <div class="flex flex-col items-center justify-center">
-                <div class="-mt-px  divide-x divide-gray-200">
+                <div class="-mt-px  divide-x divide-zinc-200">
                   <%= session.activity.description %>
                 </div>
+                <!-- FIXME: This is a placeholder image -->
                 <img src="https://picsum.photos/200/300" alt="random image" class="mt-2 w-full h-32 object-cover rounded-b-lg" />
```
Comment (MarioRodrigues10) [inline lib/atomic_web/live/department_live/show.html.heex]: This should be fixed before merged

File: lib/atomic_web/live/activity_live/index.html.heex
Comment (MarioRodrigues10) [inline lib/atomic_web/live/activity_live/index.html.heex:1]: I get an error when there's no @current_organization
image

File: lib/atomic_web/live/announcement_live/index.html.heex
Comment (MarioRodrigues10) [inline lib/atomic_web/live/announcement_live/index.html.heex:1]: I get an error when there's no @current_organization.
image

**Review by MarioRodrigues10 — DISMISSED** [review]
Make sure every page is accessible when there's no current_organization

**Comment by MarioRodrigues10** [comment]
By default, should be "Following" just like twitter and other applications.
image

**Review by joaodiaslobo — COMMENTED** [review]
image


I'm getting this error when trying to access the partner show page

**Review by joaodiaslobo — APPROVED** [review]
Outstanding work! 🚀🤘

**Pull request merged**